src/sentry/snuba/sessions_v2.py

```diff
@@ -99,7 +99,10 @@ def extract_from_row(self, row, group):
         if status == "crashed":
             return row["sessions_crashed"]
         if status == "errored":
-            return row["sessions_errored"]
+            errored_count = (
+                row["sessions_errored"] - row["sessions_crashed"] - row["sessions_abnormal"]
+            )
+            return 0 if errored_count < 0 else errored_count
```

Comment:
I would change this to `max(errored_count, 0)`

Reply:
Done :)